src/enc.cpp

```diff
@@ -24,10 +24,14 @@ using namespace std;
 using namespace ert;
 static int _argc;
 static char** _argv;
+static int envc;
+static char** envp = nullptr;
```

Comment:
```suggestion
static int _envc;
static char** _envp;
```
we use OE style in EGo's C(++) code
global pointers are nullptr by default